### ISSUE TITLE
Please, add aarch64 host support (for self-hosted runners)

### ISSUE THREAD

**Issue by iav** [crazy-max/ghaction-docker-buildx#223]
### Behaviour
On aarch64 self-hosted runner `crazy-max/ghaction-docker-buildx@v3`  tries to setupamd64 buildx version `https://github.com/docker/buildx/releases/download/v0.4.1/buildx-v0.4.1.linux-amd64`

#### Steps to reproduce this issue

1. set aarch64 self-hosted runner
2. try to run `crazy-max/ghaction-docker-buildx` on it 

#### Expected behaviour
It should run same as on x64 at least for `linux/arm64, linux/arm/v7, linux/arm/v6` targets

### Configuration

* Repository URL (if public): https://github.com/iav/test5/
* Build URL (if public): https://github.com/iav/test5/actions/runs/169293172

```yml
https://github.com/iav/test3/blob/master/.github/workflows/parallel.yml
```

### Logs
https://github.com/iav/test5/runs/871253172?check_suite_focus=true#step:3:10

**Comment by iav** [comment]
Job prepares buildx on my armbian aarch64 self-host worker:
```
      -
        name: Set up Docker Buildx
        env:
          BUILDX_URL: https://github.com/docker/buildx/releases/download/v0.4.1/buildx-v0.4.1.linux-arm64
        run: |
          curl -fsSL https://get.docker.com -o get-docker.sh
          sudo sh get-docker.sh
          mkdir -p $HOME/.docker/cli-plugins/
          curl -sL $BUILDX_URL > $HOME/.docker/cli-plugins/docker-buildx
          chmod a+x $HOME/.docker/cli-plugins/docker-buildx
          echo -e "{\n  \"experimental\": \"enabled\"\n}" | tee $HOME/.docker/config.json
          docker info
          docker buildx create --use --driver docker-container
          docker buildx inspect --bootstrap
```

**Comment by morlay** [comment]
could use tonistiigi/binfmt instead.

```
docker run --privileged --rm tonistiigi/binfmt --install all
```

**Comment by crazy-max** [comment]
@morlay Don't think so. Is runner is an aarch64 host.

**Comment by morlay** [comment]
me too.

![image](https://user-images.githubusercontent.com/1667873/91382793-15965e00-e85d-11ea-8881-e66581e48a3f.png)

**Comment by crazy-max** [comment]
@morlay Yes but buildx is already available on your host, not for @iav.

**Comment by morlay** [comment]
buildx provide arm64 version https://github.com/docker/buildx/releases
so it is easy to change for different arch. 
https://github.com/crazy-max/ghaction-docker-buildx/blob/master/src/installer.ts#L51

but i try to run `multiarch/qemu-user-static` on arm host. it failed.  `tonistiigi/binfmt` may fix this.

**Comment by morlay** [comment]
@crazy-max 

seems not released?

![image](https://user-images.githubusercontent.com/1667873/91383919-ec2b0180-e85f-11ea-8687-772a6ddc6c94.png)

**Comment by crazy-max** [comment]
@morlay 

Yes see [this workflow](https://github.com/crazy-max/test-docker-action/blob/master/.github/workflows/full.yml) for now.

**Comment by morlay** [comment]
it works well on my arm64 self-host.

![image](https://user-images.githubusercontent.com/1667873/91384460-44aece80-e861-11ea-8225-e9041378ff19.png)
https://github.com/querycap/ci-infra/actions/runs/226439751

**Comment by crazy-max** [comment]
@morlay 

> when the v1 release?

For `setup-buildx` and `setup-qemu` it will be available asap under `v0.9.0` release. No official ETA for `v1`. See https://github.com/docker/setup-buildx-action/pull/1#issuecomment-679374587

> i couldn’t wait to change my github workflows. Thanks for your wonderful work.

Thanks for your support!

**Comment by crazy-max** [comment]
@iav aarch64 support has been added to [docker/setup-buildx-action](https://github.com/docker/setup-buildx-action)
